### DISCUSSION TITLE
Animation problem?

### DISCUSSION THREAD

**Discussion by gianluca1aa** [o3de/o3de#8577] [Help Wanted]
Hi at all, I m new in o3de. I m trying to import a simple animation of a cube that move on translation and rotation axis. The import is in fbx(from 3ds max). When I put the file inside a project on o3de, I don't understand how to animate that object inside the viewport. I tried to use the "simple motion component" to view the animation in viewport but it doesnt works. How can I solve it? It seems that o3de doensn't recognized the animation, and the cube remains stationary.
Thanks in advance!

**Comment by RuneCypher** [comment]
I suggest you look at Luis Sempe video: https://www.youtube.com/watch?v=GYXG4nrQPnE

Thats helped me.
Best of luck...

**Reply by gianluca1aa** [reply]
thank you! I solved with simply attached a skin mesh to joint

**Reply by RuneCypher** [reply]
Sounds great, also  you are definitely more advanced then me. I'm new to this environment and having fun playing around with the O3DE - I just wish they had more examples that worked since I have tried to get the 3rd person animation working with no success. But haven't lost the drive to continue learning. Thanks
